smooth/components/component_battery.py

```diff
@@ -37,26 +37,35 @@
 
 Maximum chargeable/dischargeable energy
 ---------------------------------------
-The maximum chargeable and dischargeable energy [Wh] per timestep is dependant on
-the C-rate and the state of charge of the battery. Because of the charging losses
-defined by charging efficiency, the nominal value of the input flow must be higher
-than what is actually entering the battery in order to ensure that the battery can
-be fully charged in one time step. Due to the *inflow_conversion_factor*
-parameter (in :func:`~smooth.components.component_battery.Battery.create_oemof_model`),
-the battery will then receive the right amount.
+The maximum chargeable or dischargeable power [W] going in to or out of the battery is dependant
+on the C-rate and the capacity:
 
 .. math::
-    E_{in,max} = \\frac{min(c_{r,charge} \\cdot C \\cdot \\frac{t}{60},
-    C - SOC \\cdot C)}{\\mu_{charge}} \n
-    E_{out,max} = min(c_{r,discharge} \\cdot C \\cdot \\frac{t}{60}, SOC \\cdot C)
-
-* :math:`E_{in,max}` = maximum chargeable energy [Wh]
-* :math:`c_{r,charge}` = C-Rate [-/h]
-* :math:`C` = storage capacity [Wh]
-* :math:`SOC` = state of charge [Wh]
+    P_{charge,max} = E_{battery,max} \\cdot c_{r,charge}
+
+    P_{discharge,max} = E_{battery,max} \\cdot c_{r,discharge}
```

Comment:
in the description of the symbols C_{rate,(dis-)charge} is explained, I like the capitalized version more (the way I saw it in literature), moreover index r/rate does not change much but shoud be equal